app/src/controllers/pages/selectors.js

```diff
@@ -159,3 +167,74 @@ export const urlOrganizationAndProjectSelector = createSelector(
     return activeProject;
   },
 );
+
+export const activeProjectRoleSelector = createSelector(
```

Comment:
Roles are more related to the user entity rather than pages, so maybe still store them in `controllers/user`?